composer.json

```diff
@@ -17,19 +17,21 @@
         "phpstan/phpstan": "^1.10",
         "phpstan/phpstan-strict-rules": "^1.5",
         "phpunit/phpunit": "^10.4",
+        "symfony/console": "^6.3",
         "symplify/monorepo-builder": "^11.2",
         "symplify/phpstan-rules": "^12.4",
+        "t3docs/cli": "@dev",
         "typo3/coding-standards": "^0.7.1"
     },
     "replace": {
+        "t3docs/cli": "self.version",
         "t3docs/php-domain": "self.version",
         "t3docs/typo3-docs-theme": "self.version",
         "t3docs/typo3-guides-extension": "self.version"
     },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
     "autoload": {
         "psr-4": {
+            "T3Docs\\Cli\\": "packages/cli/src/",
```

Comment:
I suggest to rename the namespace, as t3docs/cli does not name the purpose correctly, perhaps something like t3docs/typo3-docs-theme-cli. What do you think?